### ISSUE TITLE
Any training results ？

### ISSUE THREAD

**Issue by bityangke** [lucasliunju/mae-colossalai#1]
Hi
Thanks for sharing.
Did you train the model？

**Comment by lucasliunju** [comment]
Yes. I have tested it for 30 epochs and it can achieve the similar accuracy as the pytorch-based mae.
